api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -1,6 +1,6 @@
 package org.pmiops.workbench.api;
 
-import static org.pmiops.workbench.billing.GoogleApisConfig.END_USER_CLOUD_BILLING;
+import static org.pmiops.workbench.google.GoogleConfig.END_USER_CLOUD_BILLING;
```

Comment:
this class still calls Google directly but I did not move this to the new CloudBillingClient because I am consideting to call Terra to get this in the following PR
https://rawls.dsde-dev.broadinstitute.org/#/users/listUserBillingAccounts